src/tools/tidy/src/walk.rs

```diff
@@ -18,6 +18,7 @@ pub fn filter_dirs(path: &Path) -> bool {
         "src/tools/clippy",
         "src/tools/miri",
         "src/tools/rust-analyzer",
+        "src/tools/rustc-perf",
```

Comment:
There's a comment above about rustfmt.toml needing to get synced as well. Can we do that?

r=me with that done.